pkg/lib/oauth/grant_offline_service.go

```diff
@@ -53,7 +53,6 @@ func (s *OfflineGrantService) IsValid(session *OfflineGrant) (bool, time.Time, e
 	return true, expiry, nil
 }
 
-// TODO(DEV-1403): Ensure the client id is correct
```

Comment:
Using OfflineGrant.Client ID is incorrect?

Reply:
We should keep using `OfflineGrant.ClientID` here to compute the expiry, but will rename `ClientID` to `InitialClientID`.